libraries/standard/http/include/http_client.h

```diff
@@ -423,19 +457,30 @@ HTTPStatus_t HTTPClient_AddRangeHeader( HTTPRequestHeaders_t * pRequestHeaders,
  * parameter pRequestBodyBuf over the transport. The response is received in
  * #HTTPResponse_t.
  *
+ * The application should close the connection with the server if any
+ * HTTP_SECURITY_ALERT_X errors are returned. Any error found in parsing is
```

Comment:
Can you state all the error codes here to be clear? Some may not understand the presence of the generic `_X`

Reply:
I do not have them all defined yet. Next PR.

Reply:
Please add a TODO for it then so that is doesn't go unnoticed